package.json

```diff
@@ -2,7 +2,7 @@
   "name": "angular2-data-table",
   "version": "0.3.8",
   "description": "angular2-data-table is a Angular2 component for presenting large and complex data.",
-  "main": "release/angular2-data-table.umd.js",
+  "main": "release/angular2-data-table.cjs.js",
```

Comment:
Yea, mentioned that in the initial comment, I have to use cjs for my project, forgot to change back after testing.